content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,95 @@ title: Custom Metrics
+- **Submitting custom metrics asynchronously**: If you want to generate custom metrics from a performance-critical Lambda function, Datadog encourage you to use Datadog Lambda Library to [submit custom metrics asynchronously](#enabling-asynchronous-custom-metrics).
+- **(Deprecated) Submitting custom metrics from CloudWatch logs**: The method to submit custom metrics by printing a log formatted as `MONITORING|<UNIX_EPOCH_TIMESTAMP>|<METRIC_VALUE>|<METRIC_TYPE>|<METRIC_NAME>|#<TAG_LIST>` has been [deprecated](#deprecated-cloudwatch-logs), and you should migrate to one of the solutions above.
+- **(Not recommended) Using a third-party library**: Most of [third-party libraries](#third-party-libraries) do not submit metrics as distributions and can lead to under-counted results.
+
+### Understanding distribution metrics
+
+Custom metrics submitted from Lambda functions are aggregated as [distributions][1], as they are designed to instrument applications, independent of the underlying hosts. You can query the metrics using aggregation `avg`, `sum`, `max`, `min`, `count`. You can also enable percentile aggregations (p50, p75, p90, p95, p99) and [manage tags][2] for aggregation on the metric summary page.
+
+Certain submission methods allow you submit metrics with timestamps from the past, up-to 20 mins old.
 
 ## Creating custom metrics from logs or traces
 
-With log-based metrics, you can record a count of logs that match a query or summarize a numeric value contained in a log, such as request duration. Log-based metrics are a cost-efficient way to summarize log data from the entire ingest stream. Learn more about creating log-based metrics [here][1]. 
+With log-based metrics, you can record a count of logs that match a query or summarize a numeric value contained in a log, such as request duration. Log-based metrics are a cost-efficient way to summarize log data from the entire ingest stream. Learn more about creating log-based metrics [here][3]. 
 
-You can also generate metrics from 100% of ingested spans, regardless of whether they are indexed by a retention filter. Learn more about creating span-based metrics [here][2].
+You can also generate metrics from 100% of ingested spans, regardless of whether they are indexed by a retention filter. Learn more about creating span-based metrics [here][4].
 
-## Sending custom metrics from the Datadog Lambda library
+## Enabling synchronous custom metrics
 
-{{< img src="serverless/serverless_custom_metrics.png" alt="Collecting Custom Metrics from AWS Lambda" >}}
+If you want to generate custom metrics from a Lambda function that is written in Python, Node.js or Golang and is performance-critical, you can use Datadog Lambda Library to submit custom metrics synchronously to Datadog API at the end your Lambda invocation.
```

Comment:
```suggestion
If you want to generate custom metrics from a Lambda function that is written in Python, Node.js or Golang and is not performance-critical, you can use Datadog Lambda Library to submit custom metrics synchronously to Datadog API at the end your Lambda invocation.
```

Reply:
I know that you meant to say "not performance-critical" here. However, even with that change, I think that makes it sound like if your Lambda is not performance-critical you **SHOULD** submit metrics synchronously, when in fact I think it would be better to use the Extension in all cases? Why would you ever want to submit metrics synchronously?